tests/components/history/test_init.py

```diff
@@ -911,3 +911,55 @@ async def test_entity_ids_limit_via_api_with_skip_initial_state(
     assert len(response_json) == 2
     assert response_json[0][0]["entity_id"] == "light.kitchen"
     assert response_json[1][0]["entity_id"] == "light.cow"
+
+
+async def test_history_with_invalid_entity_ids(
```

Comment:
Let's use `pytest.mark.parametrize` to make this a bit smaller

https://github.com/home-assistant/core/blob/d4cc4a343dd9585686e53aefb4708901ae3d5550/tests/components/sensor/test_recorder.py#L103